### ISSUE TITLE
Results were not satisfying?

### ISSUE THREAD

**Issue by alan-ai-learner** [genforce/interfacegan#79]

**Comment by fisakhan** [comment]
@alan-ai-learner can you please provide some suggestion. I also want to do similar task.

**Comment by alan-ai-learner** [comment]
> @alan-ai-learner can you please provide some suggestion. I also want to do similar task.

@fisakhan put your questions here will try to answer them.

**Comment by fisakhan** [comment]
here? where?
My question is that How can I generate multiple different images of the same identity? Different images might have different pose, expression etc. but the identity should remain the same (or look like the same identity). The implementation in this repo either changes the identity or generates an extremely poor quality image.

**Comment by alan-ai-learner** [comment]
yeah, that's the problem I faced as well, I spend like 4 months and lots of
resources on it but in the end, I haven't found the results satisfactory,
that's why I raised this issue. Btw they published one more paper called
SeFa, check that one maybe that can help.\

**Comment by fisakhan** [comment]
Thanks @alan-ai-learner
